hsextract/models/schema.py

```diff
@@ -430,10 +473,10 @@ class CoreMetadata(SchemaBaseModel):
     inLanguage: Optional[Union[LanguageEnum, InLanguageStr]] = Field(
         title="Language", description="The language of the content of the resource."
     )
-    creativeWorkStatus: Optional[Union[Draft, Incomplete, Obsolete, Published]] = Field(
+    creativeWorkStatus: Optional[Union[Draft, Incomplete, Obsolete, Published, Public, Discoverable, Private]] = Field(
```

Comment:
I think we should also remove Draft, Incomplete and Obsolete since they are not terms that we actually use, rather they are suggested possible terms given by the schema.org documentation. https://schema.org/creativeWorkStatus